File: Durwapatil/C-Programs/search-ele-in-array.c
// Date : 21 Jan 2021

// C Program to search an element in given array..

            #include<stdio.h>
            #include<stdlib.h>
            
            int main()
            {
              int a[10], i, x, p = 0, f = 0;
              printf("Enter the array elements : \n");
              for(i = 0; i < 10; i++)
              {
                scanf("%d", &a[i]);
              }
              printf("Enter the find element : \n");
              scanf("%d ", &x);
              for(i = 0; i < 10; i++)
              {
                if(a[i] == x)
                {
                  f = 1;
                  p++;
                  printf("%d no. is present in array at position %d\n", x, i);
                }
              }
              if(f == 0)
              {
                printf("Not present in array");
              }
              else
              {
                printf("%d no. is present in array %d times", x, p);
              }
              return 0;
            }

/*
Output :
    Enter the array elements : 
    1
    2
    3
    4
    5
    6
    7
    8
    9
    10
    
    Enter the find element : 
    5
    
    5 no. is present in array at position 4
    5 no. is present in array 1 times
*/

File: Gitesh2808/Java/search-ele-in-array.java
// Date : 24 Jan 2021 

// Java program to search the element in the array



import java.io.*;
class atest7
{
  public static void main(String args[])
  {
    try
    {
      DataInputStream x = new DataInputStream(System.in);
      int a[] = new int[5], i, ser;
      System.out.print("\nEnter 5 numbers : \n");
      for(i = 0; i <= 4; i++)
      {
        a[i] = Integer.parseInt(x.readLine());
      }
      System.out.print("\n Enter the search number : \n");
      ser = Integer.parseInt(x.readLine());
      for(i = 0; i <= 4; i++)
      {
        if(a[i] == ser)
        {
          System.out.print("\nNumber is present ");
          break;
        }
      }
      if(i == 5)
      {
        System.out.print("\nNumber is not present ");
      }
    }
    catch(Exception e)
    {}
  }
}


/*
Output : 
Enter 5 numbers : 
1
2
3
4
5

Enter the search number : 
3

Number is present
*/
